apps/elf-council-frontend/src/ui/gsc/GSCOverviewSection.tsx

```diff
@@ -143,9 +143,14 @@ export function GSCOverviewSection(): ReactElement {
                       />
                     </Disclosure.Button>
                     <Disclosure.Panel className="flex max-w-fit flex-col gap-3 px-4 pt-4 pb-2 text-sm text-gray-500">
-                      <p>{t`Council is an on-chain decentralized governance system through which a community can manage a DAO. It gives the community total flexibility over how to distribute Voting Power and allows it to adapt its governance system to the continuously evolving needs of the DAO.`}</p>
-                      <p>{t`The system also includes the optional structure of a Governance Steering Council (GSC) with added governance powers and responsibilities, all to be decided upon by the community.`}</p>
-                      <p>{t`This flexibility is possible thanks to the use of Voting Vaults. Learn more in `}</p>
+                      <p>{t`Any delegate who has accumulated more delegated voting
+                      power than the current GSC Eligibility Threshold (110,000
```

Comment:
Q: Do we wanna use the `formattedThreshold` variable used above for this one here too?